src/style.rs

```diff
@@ -110,6 +110,60 @@ impl fmt::Debug for Modifier {
     }
 }
 
+#[cfg(feature = "crossterm")]
```

Comment:
These might belong in crossterm.rs to avoid the feature checks and having an impl for each backend in style.rs

Reply:
Makes much more sense now that I'm writing it, to have it in crossterm with the other From impl